z3/pput.py

```diff
@@ -278,7 +278,7 @@ def parse_args():
         description='Read data from stdin and upload it to s3',
         epilog=('All optional args have a configurable default. '
                 'Order of precedence is command line args then '
-                'environment variables then user config ~/.z3.cfg'
+                'user config ~/.z3.cfg then environment variables then '
```

Comment:
There's an extra "then " at the end of this line. In the current state, this evaluates to (emphasis mine):

> Order of precedence is command line args then user config ~/.z3.cfg then environment variables **then  then** default config.

The space at the beginning of the next line should also be removed.